src/common/transformers/SPLIT.js

```diff
@@ -8,8 +8,8 @@ export const split = (value, separator) => {
     if (typeof value !== 'string') {
         throw new Error('Invalid value: need a string');
     }
-
-    return value.split(separator).map(x => x.trim());
+    let removeEmpty = value.split(separator).map(x => x.trim());
```

Comment:
A variable name, again ;)

I would have called this variable `splitted`. `removeEmpty` suggest an action, and thus a function.